.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -31,6 +31,14 @@ body:
       description: A clear description of what you expected to happen.
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: System Info
+      description: |
+        Operating system & version, npm, node, and browser(s) & version(s) (if applicable), 
+        Maybe you can try to run `npx envinfo --system --npmPackages @stencil/core --binaries --browsers` to get the necessary information
```

Can we remove this second line please? Although it's only a suggestion, I'm not comfortable asking folks to run something like this that may or may not work on their machine.
```suggestion
        Operating system & version, npm, node, and browser(s) & version(s) (if applicable)
```